Documentation/operations/troubleshooting.rst

```diff
@@ -713,6 +713,47 @@ When running in :ref:`arch_direct_routing` mode:
 
 4. Verify that the firewall on each node permits to route the endpoint IPs.
 
+Traffic to Service Endpoints is unbalanced
+------------------------------------------
+
+Symptom
+~~~~~~~
+
+Service endpoint communication is successful, but unbalanced traffic is
+observed in the following situations:
+
+  * A ``NodePort``, ``LoadBalancer`` or ``externalIP`` service is accessed from
+    outside a cluster when ``NodePort`` BPF is enabled
+  * A ``ClusterIP`` service is accessed within a cluster when the
+    :ref:`Host-reachable services<host-services>`
+
+This problem typically manifests as an imbalance of connections or requests to
+a subset of endpoints for a Service.
```

Comment:
You probably still want to mention that requests are sent from the same client which is recycling source ports.